tests/unit/apps/test_core.py

```diff
@@ -1032,3 +1036,37 @@ def test_cinder_upgrade_plan_single_unit(model):
     plan = cinder.generate_upgrade_plan(target, False, [units["cinder/0"]])
 
     assert str(plan) == exp_plan
+
+
```

Comment:
non-blocker: Maybe adding a pytest parametrize to also use `swift-proxy`

Reply:
I thought about that but I don't think there's much added value because only names of the charm/unit is changing, and no additional functionality would be tested.